plugins/azure-storage/src/components/AzureStoragePage/AzureStorageContent.tsx

```diff
@@ -125,6 +126,14 @@ export const AzureStorageContent = () => {
       folderStack.join(),
     );
   };
+  const downloadBlobDirectly = async (blobName: string) => {
+    azureStorageApi.downloadBlob(
```

Comment:
I hope this is going to the backend download only. 
```suggestion
    azureStorageApi.downloadBlobDirectly(
```